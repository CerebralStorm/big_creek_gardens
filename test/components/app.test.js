import React from 'react'
import App from '../../app/javascript/components/app'

test('render a App', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});