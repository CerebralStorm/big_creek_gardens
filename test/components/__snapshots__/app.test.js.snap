// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a App 1`] = `
<div>
  <Connect(Header) />
  <Connect(FlashMessage) />
  <Footer />
</div>
`;
